### PR TITLE
Add details to README.md

### PR DESCRIPTION
**Pull request by xjh666**
This is the first commit for the project. I only try to make changes on the README.md file, without any effect on the functionalities of other codes.

### CODE REVIEW

**Comment by luckyman20** [comment]
@xjh666 Pull request could have some better title which describes the changes you are making.

File: README.md [hunk 1]
```diff
@@ -18,6 +24,10 @@ wget --no-check-certificate -P fineract-provider/gradle/wrapper https://github.c
 
 curl --insecure -L https://github.com/apache/fineract/raw/develop/fineract-provider/gradle/wrapper/gradle-wrapper.jar > fineract-provider/gradle/wrapper/gradle-wrapper.jar
 
+Instructions to download MySQL
+============
+If MySQL is not installed in the device, download from [MySQL](https://www.mysql.com/downloads/ "MySQL").
+
```
Comment (vishwasbabu) [inline README.md:30]: Maybe add details of the MySQL version required like you have for Java ? Else, remove this section and add a hyperlink directly to the requirements section itself, as there isn't much information here to warrant the addition of another section.

**Review by vishwasbabu — CHANGES_REQUESTED** [review]
comments inline

**Comment by vorburger** [comment]
#expired

**Pull request closed without merging**